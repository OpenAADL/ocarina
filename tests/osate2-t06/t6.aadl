package t6
public
new_classifier_name renames thread pkg::old_classifier_name;
system system_t6
end system_t6;
end t6;