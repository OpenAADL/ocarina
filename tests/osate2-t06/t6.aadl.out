t6.aadl:3:41: pkg::old_classifier_name (identifier) is not visible
Cannot analyze AADL specifications
